static/css/style.css

```diff
@@ -45,6 +58,117 @@ form {
       background-color: #b07337;
       color: #000; }
```

Comment:
Please use a variable for color: #000; Please check the available variables defined in base.scss